### ISSUE TITLE
[ui5-combobox ]: a11y screen reader does not announce selection correctly when selecting a value

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents#10434]
### Bug Description

Go to: https://sap.github.io/ui5-webcomponents/components/ComboBox/
Select Austria
Screen Reader NVDA does not announce the selection.

![Image](https://github.com/user-attachments/assets/df522ed3-1407-42e6-a5d3-b68f7f81e22d)

https://sap.github.io/ui5-webcomponents/components/ComboBox/

![Image](https://github.com/user-attachments/assets/7a88364a-e74c-4194-ae21-0d21b5f1fd0f)

Now Select Germany. Screen Reader announces "Germany unselected"

![Image](https://github.com/user-attachments/assets/483706b0-cf47-4821-9130-594731b121e7)

### Expected Behaviour

Screen Reader should announce the selections as they are selected or changed.

### Steps to Reproduce

1.
2.
3.
...

### Priority

Medium

### UI5 Web Components Version

2.x

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by ivoplashkov** [comment]
Hello colleagues,

The issue is reproduced as described by the reporter. Sometimes NVDA does not announce the selection, but this happens only using mouse click. When navigating through the items with keyboard, the selection is announced, however once you select an item screen reader announced "unselected".

Could you please have a look?
Best regards,
Ivaylo

**Comment by elenastoyanovaa** [comment]
Hello @gurkirpalgill ,

The announcement of selected/unselected actually comes from the action of selecting/unselecting text in the input field and has nothing to do with the suggestions. Suggestions are announced as selected if you navigate with keyboard through the items. For the "unselected" once you choose an item, the selection of the text (with blue background) gets unselected - which is observed in the screenshot you provided and the Screen Reader vendor reacts to this action and announces it to the user. This is a behavior of the screen reader software and it is reproducible with native html as well. We cannot prevent this speech output of happening. 

Kind Regards,
Elena
